generators/generator-base-private.js

```diff
@@ -1550,9 +1550,9 @@ module.exports = class extends Generator {
         // Prettier is clever, it uses correct rules and correct parser according to file extension.
         let prettierFilter;
         if (this.prettierJava) {
-            prettierFilter = filter(['.yo-rc.json', '{,**/}*.{md,json,ts,tsx,scss,css,yml,java}'], { restore: true });
+            prettierFilter = filter(['.yo-rc.json', '.jhipster/*.json', '{,**/}*.{md,json,ts,tsx,scss,css,yml,java}'], { restore: true });
```

Comment:
Does this expression includes `.jhipster/modules/jhi-hooks.json` ? I think it should be formatted too.
The following expression might be a bit more inclusive:
```suggestion
            prettierFilter = filter(['.yo-rc.json', '{,**/,.jhipster/**}*.{md,json,ts,tsx,scss,css,yml,java}'], { restore: true });
```